tensorrt_llm/_torch/pyexecutor/model_engine.py

```
@@ -456,17 +461,39 @@ def use_mrope(self):
 
+    @property
+    def is_warmup(self):
+        return getattr(self, "_is_warmup", False)
+
+    @is_warmup.setter
+    def is_warmup(self, value: bool):
+        self._is_warmup = value
+        if self._torch_compile_enabled:
+            if value:
+                self._torch_compile_backend.enable_optimization()
+                set_enable_piecewise_cuda_graph_capture_flag(True)
```

Ruff (0.12.2)

Undefined name `set_enable_piecewise_cuda_graph_capture_flag`

(F821)